cmake/Dependencies.cmake

```diff
@@ -91,7 +91,7 @@ if(BUILD_BENCHMARK)
 endif()
 
 # Find or download/install rocm-cmake project
-find_package(ROCM QUIET CONFIG PATHS /opt/rocm)
+find_package(ROCM QUIET CONFIG PATHS ${CMAKE_INSTALL_PREFIX})
 if(NOT ROCM_FOUND)
```

Comment:
Does this impact the build and install documentation for this component?

What happens when this component package is built without CMAKE_INSTALL_PREFIX and installed in a multi ROCm installed system?

Reply:
These changes should not affect the default build steps of the rocPRIM build. 
The changes are mainly to build a rocPRIM package so that it can be built and installed into NON /opt/rocm path. 
CMAKE_INSTALL_PREFIX variable default value is /opt/rocm, so it default to /opt/rocm.
In case of multiple ROCm installation, say we have Eg :/opt/rocm-2.8.1, /opt/rocm-2.7.4, /opt/rocm-2.10.1, then in case 
Case 1: CMAKE_INSTALL_PREFIX should be set the rocm version (folder path) where to get the rocm packages like hip/hcc and rocPRIM package will be built to be installed inside this location.
Case 2: Say user want to use hcc/hip from particular version(eg: rocm-2.8.1) of rocm but rocPRIM package to be installed into a different location (/usr/local/rocm-math/)
- Set CMAKE_INSTALL_PREFIX=/usr/local/rocm-math/
- Set a symlink for rocm version #ln -s /opt/rocm-2.8.1 /opt/rocm

Reply:
What if /opt/rocm is NOT a symlink and has rocm2.7 version installed? Would the "set a symlink for rocm version" step in Case 2 require previous rocm version to be uninstalled? 
Might want to add your explanations/use cases to the README also.

Still needs some clarification on: "What happens when this component package is built without CMAKE_INSTALL_PREFIX and installed in a multi ROCm installed system?" Since the default value is /opt/rocm, a default build of this package would build it to be installed under /opt/rocm even in a multiple ROCm installation system. Should that be permitted, given that that would overwrite any /opt/rocm symlink or create /opt/rocm? Could install scripts be updated to check for multiple rocm /opt/rocm-<ver> installation and fail installation of default /opt/rocm packages?